test/testcafe/framework/page-objects/ChallengeFactorPageObject.js

```diff
@@ -27,6 +27,10 @@ export default class ChallengeFactorPageObject extends BasePageObject {
     return Selector(SWITCH_AUTHENTICATOR_SELECTOR).textContent;
   }
 
+  clickSwitchAuthenticatorButton() {
```

Comment:
nit: do we need to add this here and in FactorEnrollPasswordPageObject.js? It's already in BaseFormObject.js right?

Reply:
I think I can move these to BasePageObject so that I dont need to add it here and on FactorEnrollPasswordPageObject